From 15ffab731c9b0a5e70890078ebaf1031d64495d9 Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date: Sat, 19 Nov 2011 15:17:52 -0500
Subject: [PATCH 336/757] Document that depmod needs to be executed by hand

External modules don't seem to actually execute depmod on make install,
even though the console output says so. Document that it needs to be
manually executed in the README file.

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
---
 drivers/staging/lttng/README | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/staging/lttng/README b/drivers/staging/lttng/README
index 3bd0d15..f4e3368 100644
--- a/drivers/staging/lttng/README
+++ b/drivers/staging/lttng/README
@@ -22,12 +22,14 @@ access to your full kernel source tree), and use:
 
 % make
 # make modules_install
+# depmod -a
 
 If you need to specify the target directory to the kernel you want to build
 against, use:
 
 % KERNELDIR=path_to_kernel_dir make
 # KERNELDIR=path_to_kernel_dir make modules_install
+# depmod -a kernel_version
 
 Use lttng-tools to control the tracer. LTTng tools should automatically load
 the kernel modules when needed. Use Babeltrace to print traces as a
-- 
2.0.2

